market/connector/adobe-esign-connector/meta.json

```diff
@@ -0,0 +1,22 @@
+{
+    "$schema": "https://json-schema.axonivy.com/market/10.0.0/meta.json",
+    "id": "adobe-esign-connector",
+    "name": "Adobe Esign Connector",
```

Comment:
based on [Wikipedia](https://en.wikipedia.org/wiki/Adobe_Sign), the official name is "Adobe Acrobat Sign", please change the name to "Adobe Acrobat Sign"

Reply:
Hi @andreasbalsiger,
Do we need to change the name of repo, package and other to "Adobe Acrobat Sign" also?

Reply:
I am puristic so I would say yes... but I am not aware if this causes additional problems we dont want. Maybe @ivy-sgi , you can share your opinion here?

Reply:
Hi @andreasbalsiger ! Changing the repo name is a bit dangerous and we are not 100 sure if we know all dependencies. How worse would it be for you to approve this PR as it is?

Reply:
Hi @ivy-sgi : Fair enough... lets approve it like this!